src/lib/blueprint.ts

```diff
@@ -909,7 +919,14 @@ export const getSemanticMethod = (methods: Method[]): Method => {
   }
 
   const priorityOrder: Method[] = ['PUT', 'PATCH', 'POST', 'GET', 'DELETE']
-  return methods.find((m) => priorityOrder.includes(m)) ?? 'POST'
```

Comment:
This seems to have been implemented incorrectly because the priority was not defined by the `priorityOrder` but by the passed `methods`. 
Though now, tests are failing. Am I understanding this incorrectly? 
Is the `priorityOrder` to adjust the tests if I am right that there's a bug here.

This seems to have been implemented incorrectly because the priority was defined by the passed `methods`, not by the `priorityOrder`.
However, now the tests are failing. Am I understanding this incorrectly?
If I'm right that there's a bug here, is the `priorityOrder` correct so that I could adjust the tests?

Reply:
This does looks like a mistake 🤔

Reply:
@razor-x I change the priority method in this commit 8f9e78fab7ece5a1a3105d2aaeec30ce218adfad to fix the tests, does it look right?